recipe_scrapers/_exceptions.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, domain):
 
 
 class NoSchemaFoundInWildMode(RecipeScrapersExceptions):
-    """Error when wild_mode fails to locate schema at the url"""
+    """Error when wild_mode fails to locate schema at the URL"""
```

Comment:
This seems to duplicate the name of the exception; we might be able to make better use of the screen space and reading time.

```suggestion
    """The scraper was unable to locate schema.org metadata within the webpage."""
```

(also, sometimes if reading some content in a foreign language that you only understand to a limited level, it can help if the same idea is repeated in two slightly different ways)